graph2tac/loader/predict_server.py

```diff
@@ -285,10 +284,14 @@ def _enter_coq_context(self, definitions: OnlineDefinitionsReader, tactics):
 
         self.data_server.align_definitions(definitions.definitions())
 
-        logger.info(f"initializing network with {len(self.data_server.global_defs)} defs in global context")
+        if len(self.data_server.global_defs) > 0:
+            static_global_context = np.arange(max(self.data_server.global_defs)+1, dtype=np.uint32)
+        else:
+            static_global_context = np.arange(1, dtype=np.uint32)
+        logger.info(f"initializing network with {static_global_context} defs in global context")
```

Comment:
Should this be `{len(static_global_context)}` like it used to be.  I think for an info-level logging statement, listing all the indices in the global context would be too much.